articles/service-fabric/service-fabric-backup-restore-service-ondemand-backup.md

```diff
@@ -87,9 +87,12 @@ Use the [BackupPartition](/rest/api/servicefabric/sfclient-api-backuppartition)
 
 ```powershell
 $StorageInfo = @{
-    ConnectionString = 'DefaultEndpointsProtocol=https;AccountName=<account-name>;AccountKey=<account-key>;EndpointSuffix=core.windows.net'
-    ContainerName = 'backup-container'
-    StorageKind = 'AzureBlobStore'
+    StorageKind = "ManagedIdentityAzureBlobStore"
+    FriendlyName = "AzureMI_storagesample"
+    BlobServiceUri = "https://<account-name>.blob.core.windows.net"
+    ContainerName = "backup-container"
+    ManagedIdentityType = "VMSS"
+    ManagedIdentityClientId = "<clien-ID of User-Assigned MI>"  # Use Optional parameter `ManagedIdentityClientId` with clien-ID of User-Assigned Managed Identity in case of multiple User-Assigned managed identities assigned to your resource, else no need of this paramter.
```

Comment:
Same needs to be clear when to use ManagedIdentityClientId

Reply:
updated